velox/common/memory/MappedMemory.h

```diff
@@ -147,20 +157,70 @@ class MappedMemory {
 
     // Returns the run number and the position within the run
     // corresponding to 'offset' from the start of 'this'.
-    void findRun(uint64_t offset, int32_t* index, int32_t* offsetInRun);
+    void findRun(
```

Comment:
Any particular reason to pass index and offsetInRun as raw pointers instead of returning a tuple?